CHANGELOG.rst

```diff
@@ -5,6 +5,10 @@ in development
 --------------
 * Exit hubot on unhandled promise rejections and uncaught exceptions, usually caused by unrecoverable errors (bug fix)
```

Comment:
@jinpingh Because this change was already in `master`, changelog record should be moved under `0.9.4` during the merge with master/rebase process.

Reply:
Please pay more attention next time when syncing with `master`.

Reply:
cc @blag

Reply:
This one is on me - I noticed that and didn't change it, since I figured the issue that it is a part of is not quite "all the way" fixed. As far as I'm aware though, the changelog is only going to be read by humans, so the changelog being "wrong" isn't going to break any software.

I will be more strict about this in the future.